js/client-teambuilder.js

```diff
@@ -463,11 +463,10 @@
 			}
 		},
 		defaultDragEnterTeam: function (e) {
-			// Drag originated outside teambuilder
-			if (!e.originalEvent.dataTransfer) return;
-			if (!e.originalEvent.dataTransfer.files) return;
-			if (e.originalEvent.dataTransfer.files[0] && e.originalEvent.dataTransfer.files[0].name.slice(-4) !== '.txt') return;
```

Comment:
How can I reproduce that? I tried dragging to the desktop client (on Windows XP) and `dataTransfer.files` was an empty list, as expected, at this point.